apps/progs.h

```diff
@@ -225,6 +225,12 @@ static FUNCTION functions[] = {
 #ifndef OPENSSL_NO_RMD160
     { FT_md, "rmd160", dgst_main},
 #endif
+#ifndef OPENSSL_NO_BLAKE2B
```

Comment:
`OPENSSL_NO_BLAKE2B` and `OPENSSL_NO_BLAKE2S` will never be defined as far as I can see, so it's not just unnecessary, these conditions are faulty